docs/_lab/05-writing-tasks/index.md

```diff
@@ -94,20 +96,20 @@ Ran on 1 node in 3.87 seconds
 * As with the Bash example above, name parameters so that they're more easily understood by users of the task.
 * By default tasks with a `.ps1` extension executed over WinRM use PowerShell standard agrument handling rather than being supplied as prefixed environment variables or via `stdin`. 
 
-# Write your first task in Python
+## Write Your First Task in Python
 
-Note that Bolt assumes that the required runtime is already available on the target nodes. For the following examples to work, Python 2 or 3 must be installed on the target nodes. This task will also work on Windows system with Python 2 or 3 installed.
+When running a task, Bolt assumes that the required runtime is already available on the target nodes. For the following examples to work, Python 2 or 3 must be installed on the target nodes. This task will also work on Windows system with Python 2 or 3 installed.
 
-Save the following as `modules/exercise5/tasks/gethost.py`:
+Save the following as `Boltdir/site-modules/exercise5/tasks/gethost.py`:
 
 ```python
 {% include_relative modules/exercise5/tasks/gethost.py -%}
 ```
 
-Run the task using the command `bolt task run <task-name> <task options>`.
+Run the task using the command `bolt task run <task-name>`.
 
 ```shell
-bolt task run exercise5::gethost host=google.com --nodes all --modulepath ./modules
+bolt task run exercise5::gethost host=google.com --nodes node1
```

Comment:
Does `all` not work here?

Reply:
The task fails on the Windows node - I think I had changed this to just `node1` prior to creating the `linux` group in the inventory and just missed changing it back.